apps/web/config-overrides.js

```diff
@@ -1,8 +1,16 @@
 const { useBabelRc, override } = require('customize-cra');
+const { DefinePlugin } = require('webpack');
+const { version } = require('./package.json');
 // const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 
 function overrideConfig(config, env) {
-  const plugins = [...config.plugins /* new BundleAnalyzerPlugin() */];
+  const plugins = [
+    ...config.plugins,
+    new DefinePlugin({
+      'process.env.NOVU_VERSION': JSON.stringify(version),
+    }),
```

Comment:
it will replace `process.env.NOVU_VERSION` with version string.

We need JSON.stringify because Webpack will put this value into bundle as is without string quotes.